### DISCUSSION TITLE
Twitter Cards Not Displaying Properly

### DISCUSSION THREAD

**Discussion by JordanLeviLV** [mmistakes/so-simple-theme#414] [Q&A] [answered]
OK, so i got my site mostly set up now, but I noticed that links display properly on facebook, but not twitter. i've tried multiple things, but can't figure it out. i even tried out the code that's listed under the twitter cards section of the old readme, but no luck. i noticed the links from your example website display properly on twitter tho, so can anyone help me figure out what I'm doing wrong? my site's https://jordanrlevi.github.io/ , thanks again for the great theme!

**Comment by mmistakes** [comment, marked as answer]
Might be your paths. View source on any of your post/pages and you'll see a double `//`. That might be confusing Twitter.
I think you'll want to remove the end slash in [your `url`](https://github.com/JordanRLevi/jordanrlevi.github.io/blob/master/_config.yml#L23) defined in _config.yml

![Screen Shot 2021-02-24 at 3 24 08 PM](https://user-images.githubusercontent.com/1376749/109061248-6c69dc80-76b4-11eb-9f12-6105cd94b770.png)

Twitter's card validator is a good place to start to for debugging. I'm not seeing any errors but it's not showing the image, which leads me to believe it doesn't like your paths.

https://cards-dev.twitter.com/validator

**Reply by JordanLeviLV** [reply]
AAAAH! ok, my bad for trying so late, but i've been busy af the past couple days, but getting rid of the dash in the url in the config worked!!!!!! omg, that was so mf frustrating lmao thank you so much!
